src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -28,7 +28,8 @@
 SSLERROR_TEMPLATE = ('Certificate verification failed. This typically happens when using Azure CLI behind a proxy '
                      'that intercepts traffic with a self-signed certificate. '
                      # pylint: disable=line-too-long
-                     'Please add this certificate to the trusted CA bundle. More info: https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#work-behind-a-proxy.')
```

Comment:
`en-us` is removed so that the doc website (https://docs.microsoft.com/) can automatically choose the language for the user.